docs/network_discovery.md

```diff
@@ -9,6 +9,16 @@ Home Assistant has built-in helpers to support mDNS/Zeroconf and SSDP. If your i
 
 ## Bluetooth
 
+### Best practices
```

Comment:
This is mainly for 3rd party library authors, right?

Reply:
Yes